CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing to the Cloud Azure Ops Ansible Collection
+ 
+We welcome community contributions to this collection.
+This guide provides clear instructions for contributors on how to set up their environment for contributing to the `cloud.azure_ops` Ansible Collection.
+If you find problems, please open an issue or create a PR against this collection repository.
+
+## Prerequisites
+
+Make sure you have the following installed on you Machine:
+
+1. Python3 and pip3
+2. Docker or Podman
```

Comment:
I didn't succeeded to run tests without Docker install

Reply:
Did you have podman installed and it didn't work?

Reply:
yes

Reply:
Following to ansible documentation it should work
https://docs.ansible.com/ansible/latest/dev_guide/testing_running_locally.html#containers

Reply:
Its seems in the Ansible docs that Podman should be supported
I guess you have local/configuration issue
I will try to run with podman and will let you know, but this is not related to this PR